<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="0" failed="1" skipped="2">
  <reporter-output>
    <line>
      <![CDATA[login unsuccessful]]>
    </line>
  </reporter-output>
  <suite started-at="2023-07-23T14:51:54 IST" name="Default suite" finished-at="2023-07-23T14:52:16 IST" duration-ms="22323">
    <groups>
    </groups>
    <test started-at="2023-07-23T14:51:54 IST" name="Default test" finished-at="2023-07-23T14:52:16 IST" duration-ms="22323">
      <class name="PetStoreTest.JpetStoreTest">
        <test-method is-config="true" signature="initialize()[pri:0, instance:PetStoreTest.JpetStoreTest@1ffaf86]" started-at="2023-07-23T14:51:54 IST" name="initialize" finished-at="2023-07-23T14:52:06 IST" duration-ms="11835" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method signature="loginTest()[pri:1, instance:PetStoreTest.JpetStoreTest@1ffaf86]" started-at="2023-07-23T14:52:06 IST" name="loginTest" finished-at="2023-07-23T14:52:16 IST" duration-ms="9909" status="FAIL">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.fail(Assert.java:96)
at PetStoreTest.JpetStoreTest.loginTest(JpetStoreTest.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[login unsuccessful]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method signature="searchTest()[pri:2, instance:PetStoreTest.JpetStoreTest@1ffaf86]" started-at="2023-07-23T14:52:16 IST" name="searchTest" finished-at="2023-07-23T14:52:16 IST" depends-on-methods="PetStoreTest.JpetStoreTest.loginTest" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method PetStoreTest.JpetStoreTest.searchTest() on instance PetStoreTest.JpetStoreTest@1ffaf86 depends on not successfully finished methods 
[PetStoreTest.JpetStoreTest.loginTest() on instance PetStoreTest.JpetStoreTest@1ffaf86]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method PetStoreTest.JpetStoreTest.searchTest() on instance PetStoreTest.JpetStoreTest@1ffaf86 depends on not successfully finished methods 
[PetStoreTest.JpetStoreTest.loginTest() on instance PetStoreTest.JpetStoreTest@1ffaf86]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchTest -->
        <test-method signature="addCartTest()[pri:3, instance:PetStoreTest.JpetStoreTest@1ffaf86]" started-at="2023-07-23T14:52:16 IST" name="addCartTest" finished-at="2023-07-23T14:52:16 IST" depends-on-methods="PetStoreTest.JpetStoreTest.loginTest" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method PetStoreTest.JpetStoreTest.addCartTest() on instance PetStoreTest.JpetStoreTest@1ffaf86 depends on not successfully finished methods 
[PetStoreTest.JpetStoreTest.loginTest() on instance PetStoreTest.JpetStoreTest@1ffaf86]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method PetStoreTest.JpetStoreTest.addCartTest() on instance PetStoreTest.JpetStoreTest@1ffaf86 depends on not successfully finished methods 
[PetStoreTest.JpetStoreTest.loginTest() on instance PetStoreTest.JpetStoreTest@1ffaf86]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:156)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addCartTest -->
        <test-method is-config="true" signature="closeDriver()[pri:0, instance:PetStoreTest.JpetStoreTest@1ffaf86]" started-at="2023-07-23T14:52:16 IST" name="closeDriver" finished-at="2023-07-23T14:52:16 IST" duration-ms="467" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeDriver -->
      </class> <!-- PetStoreTest.JpetStoreTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
